metrics/errors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'enables JavaScript Sentry error metrics',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uuid', 'sampleRate'],
+  independentVariables: ['sentryEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! subject.sampleRate && subject.uuid && this.bernoulliTrial(subject.sampleRate, subject.uuid)
```

Comment:
"hardcode" is a relative term. What I think we really mean when we talk about hardcoding or not is "when is this value bound?" I *think* you'd rather put it in "configuration" because it seems easier to change or not coupled to code, hence softer. But in this case experiments are the softest code (or config) in the system. Can we deploy a content-server config change or an experiment change faster? :)

Don't worry about what file a value is set in, worry about binding the value at the right time and in the right context.